devimint/src/external.rs

```diff
@@ -915,6 +915,14 @@ pub async fn open_channels_between_gateways(
 
     bitcoind.mine_blocks(10).await?;
 
+    debug!(target: LOG_DEVIMINT, "Syncing gateway lightning nodes to chain tip...");
```

Comment:
Note for reviewers: We're adding this before waiting for channel readiness because it now triggers a chain sync, where as waiting for channel readiness does not.